hosts

```diff
@@ -146,15 +146,30 @@ pulsar_qld_blast
 pulsar_nci_training_head
 pulsar_azure_0_head
 
+# All pulsar head nodes
+[pulsar_head_nodes:children]
+production_pulsars
+dev_pulsar_head
+staging_pulsar_head
+pulsar_nci_test_head
+
+# All pulsar workers
+[pulsar_workers:children]
+pulsar_paw_workers
```

Comment:
Isn't pulsar-paw gone?

Reply:
Thanks JL! I need to add pulsar_qld_gpus to production_pulsars as well